### PR TITLE
MTM-46186: Clarify messaging release support

### PR DESCRIPTION
**Pull request by sagscmi**
Clarify in which releases the Messaging Service and features built on it are available.  Will unfortunately need to be grafted back as far as 10.11.

### CODE REVIEW

File: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md [hunk 1]
```diff
@@ -8,9 +8,16 @@ aliases:
...
-{{< /c8y-admon-important >}}
+{{< c8y-admon-info >}}
+The {{< product-c8y-iot >}} Messaging Service is an optional component of the {{< product-c8y-iot >}} platform that may need to be enabled before the microservice-based data broker can be used.
+The original data broker will continue to operate alongside the microservice-based data broker for the time being, and users can choose which data broker to use on a per-tenant basis.
+
+For the shared public cloud instances of the {{< product-c8y-iot >}} platform, the Messaging Service is enabled by default on release 10.13 and above, and the microservice-based data broker can be enabled on requst for individual tenants that already have access to the original data broker.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/ms-data-broker.md:15]:
```suggestion
For the shared public cloud instances of the {{< product-c8y-iot >}} platform, the Messaging Service is enabled by default on release 10.13 and above, and the microservice-based data broker can be enabled on reqeust for individual tenants that already have access to the original data broker.
```

File: content/reference/notifications-bundle/overview.md [hunk 2]
```diff
@@ -4,18 +4,26 @@ title: Overview
...
+{{< /c8y-admon-info >}}
+
+Notifications 2.0 improves upon the [Real-time notification API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API) by providing stronger delivery semantics and ordering guarantees.
+It is also intended to be simpler to use than the "Bayeaux" protocol used in the Real-time notification API.
+New capabilities are added to Notifications 2.0 in each release of {{< product-c8y-iot >}}.
+However, it does not yet support all of the notifications available from the Real-time notification API so it is not yet a complete replacement for the older API.
+See the rest of this section and the detailed API documentation for full details of the notifications supported by this release of the Notifications 2.0 API.
+
+To receive notifications over the Notifications 2.0 protocol, an application or microservice must subscribe to notifications, either for notifications about a particular managed object or in a wider context that is scoped to a tenant. Subscriptions are persistent,long lived, and allow notifications to be stored reliably until consumed and acknowledged by a consuming microservice or application.
```
Comment (gapa-c8y) [inline content/reference/notifications-bundle/overview.md:26]:
```suggestion
To receive notifications over the Notifications 2.0 protocol, an application or microservice must subscribe to notifications, either for notifications about a particular managed object or in a wider context that is scoped to a tenant. Subscriptions are persistent, long lived, and allow notifications to be stored reliably until consumed and acknowledged by a consuming microservice or application.
```

**Review by gapa-c8y — CHANGES_REQUESTED** [review]

File: content/reference/notifications-bundle/overview.md [hunk 3]
```diff
@@ -4,18 +4,26 @@ title: Overview
...
-Currently, device management notifications equivalent to what is covered by real-time notifications are available, but delivered with at-least-once semantics over a new protocol.
-In the future, further platform events will be included as 2.0 notifications.
+The [Notifications 2.0 API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Notification-2.0-API) allows applications or microservices to receive and process notifications generated by the use of the {{< product-c8y-iot >}} REST or MQTT APIs (device management, measurements, alarms, events and other platform APIs) in a reliable manner.
 
-This new Notifications 2.0 API supersedes the [Real-time notification API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API), as it provides for stronger delivery semantics and ordering.
-It is also intended to be simpler to use than the Bayeaux protocol used in the Real-time notification API.
+{{< c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/overview.md:9]: @sagscmi Would it make sense to use the "requirements" admonition here {{< c8y-admon-req >}} and further below? As this is what it is about, isn´t it?
{{< c8y-admon-req >}}

Comment (sagscmi) [reply content/reference/notifications-bundle/overview.md:9]: @BeateRixen Yes that makes sense, I will make the change in both files.  I actually didn't know that tag existed, and I didn't want to use the `important` admonition, which seemed a bit too aggressive.

**Review by BeateRixen — APPROVED** [review]

**Review by gapa-c8y — APPROVED** [review]

**Pull request merged**